src/expression/node/FunctionNode.js

```diff
@@ -57,6 +57,13 @@ export const createFunctionNode = /* #__PURE__ */ factory(name, dependencies, ({
 
   FunctionNode.prototype.isFunctionNode = true
 
+  /* Returns length-limited string of its argument */
+  function strin (entity) {
```

Comment:
Can you move this util function into `./src/utils/string.js`, and maybe make it a bit more generic, like :

```js
export function truncate(text, limit = 100) {
  // ...
}

Reply:
OK, when I got to utils/string.js (which I had not yet realized existed) it seemed to make more sense just to put a truncate option on format, which it seemed like I should have been using anyway. So I've just pushed a commit (and rebased on develop) to do that.

Reply:
Ah, I see you extended `format`. That's neat 👍